### PR TITLE
[remux] update remux with shutdown fix for rM1

### PR DESCRIPTION
**Pull request by raisjn**
the new rM update (2.5) removes rtcwake from the system. this version of remux no longer depends on rtcwake

### CODE REVIEW

**Comment by Eeems** [comment]
It would be great to start shipping rtcwake as a package since it's got a nicer interface

File: package/rmkit/package [hunk 1]
```diff
@@ -62,7 +62,7 @@ nao() {
 remux() {
     pkgdesc="App launcher that supports multi-tasking applications"
     url="https://rmkit.dev/apps/remux"
-    pkgver=0.1.0-1
+    pkgver=0.1.0-2
```
Comment (Eeems) [inline package/rmkit/package]: No version number change?

Comment (raisjn) [reply package/rmkit/package]: i don't have internal version numbers for programs in rmkit,  I'll add that to the todo list

**Comment by Eeems** [comment]
The other sub-packages should have their release number bumped to ensure that they will register as an update.
When trying to launch remux it's turning off my device. https://pastebin.com/n0866iKF

**Comment by Eeems** [comment]
harmony, mines, and nao/simple all seem to work fine though.

**Review by Eeems — APPROVED** [review]

**Pull request merged**